src/components/settings-row/settings-row.d.ts

```diff
@@ -1,10 +1,13 @@
 import * as React from "react";
 
+type HeadingTypes = "h1" | "h2" | "h3" | "h4" | "h5" | "h6";
```

Comment:
we should import this from the `Heading` component in my opinion, as there's no reason to suppose it will ever be different.

And if there is a good reason to redefine it here then the same comment applies as I made there: the name should be singular (`HeadingType`).